tests/DCPS/MultiTopic/MultiTopicTest.cpp

```diff
@@ -7,25 +7,305 @@
 #include <dds/DCPS/PublisherImpl.h>
 #include <dds/DCPS/SubscriberImpl.h>
 #include <dds/DCPS/WaitSet.h>
-#include "dds/DCPS/StaticIncludes.h"
+#include <dds/DCPS/StaticIncludes.h>
+#include <dds/DCPS/transport/framework/TransportRegistry.h>
+#include <dds/DCPS/SafetyProfileStreams.h>
 
-#include "dds/DCPS/transport/framework/TransportRegistry.h"
-
-#include "MultiTopicTestTypeSupportImpl.h"
+#include <MultiTopicTestTypeSupportImpl.h>
 
 #include <sstream>
 #include <stdexcept>
 
 using namespace DDS;
 using namespace OpenDDS::DCPS;
 
+/*
+ * These wrappers make it possible to use this source code with both the
+ * classic C++ mapping and the C++11 mapping
+ */
+#ifdef CLASSIC_MAPPING
+bool operator!=(const TAO::String_Manager& a, const TAO::String_Manager& b)
+{
+  return std::strcmp(a.in(), b.in());
+}
+
+bool sequences_are_not_equal(const Airports& a, const Airports& b)
+{
+  const unsigned len = a.length();
+  if (len != b.length()) {
+    return true;
+  }
+  for (unsigned i = 0; i < len; i++) {
```

Comment:
```suggestion
  for (unsigned i = 0; i < len; ++i) {
```

Reply:
What is the benefit for doing this for integers?

Reply:
It is in the guidelines.  I believe this is common with ACE as well.  Rationale is that if the developer is in the habit of always writing it prefix (when the expression result is unused), it will be right without having to think about it.  It is also a case of the general principle of using the least complex operation that meets the requirements.